packages/flutter_tools/templates/app_shared/windows.tmpl/CMakeLists.txt.tmpl

```diff
@@ -1,13 +1,16 @@
+# Project-level configuration.
 cmake_minimum_required(VERSION 3.14)
 project({{projectName}} LANGUAGES CXX)
 
+# The name of the executable created for the application. Change this to change
+# the on-disk name of your application.
 set(BINARY_NAME "{{projectName}}")
 
+# Explicitly opt in to modern CMake behaviors to avoid warnings with recent
+# versions of CMake.
 cmake_policy(SET CMP0063 NEW)
 
-set(CMAKE_INSTALL_RPATH "$ORIGIN/lib")
```

Comment:
This is the only non-comment change; see https://github.com/flutter/flutter/issues/93930